client/components/Reviews/__test__/Review.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import EnzymeAdapter from 'enzyme-adapter-react-16';
+
+import Reviews from '../components/Reviews.js';
+import Ratings from '../components/Ratings.js';
+import ReviewsAndRatings from '../index.js';
+
+// set up enzyme's react adapter
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+
+/*
+test('Review Component Exists', () => {
```

Comment:
try to erase comments before you commit I did the same thing on a few of my last commits

Reply:
@melliottgithub deleted the comments